src/2048-expert/index.html

```diff
@@ -13,6 +13,22 @@
 <body>
     <core-navbar>2048 Expert</core-navbar>
 
+    <!-- Modal window with instructions on how to play -->
+    <core-modal>
+        <div class="modal-header">HOW TO PLAY 2048 Expert</div>
+        <p class="modal-text">
+        <ul>
+            <li>Take actions with arrow keys</li>
```

Comment:
"Use the arrow keys to move the tiles" might be a more clear instruction for the user